cli.js

```diff
@@ -23,6 +23,7 @@ var core = require('./lib/core');
 var bundle = require('./lib/bundle');
 var endpoint = require('./lib/endpoint');
 var install = require('./lib/install');
+var cache = require('./lib/cache');
```

Comment:
Rather than having a dedicated module for `cache`, lets put this in `core`. I don't think we'll have other cache functions? Also `cache` is not something to think of as a single interface as well.